src/com/sindhu/webapplication/UpdateItem.java

```diff
@@ -13,8 +13,8 @@ public static void updateItem(String itemName , int id) {
             if (connection == null) {
                 return;
             }
-            Statement statement = connection.createStatement();
-            statement.executeUpdate(update);
+            PreparedStatement preparedStatement = connection.prepareStatement(update);
```

Comment:
move `updateItem` function to `ToDoList.java`. and rename it to `updateToDoItem`